samples/bmp280-sensor-azure-iot-hub/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Bmxx80;
+using Iot.Device.Common;
+using UnitsNet;
+using System.Device.Gpio;
+using Microsoft.Azure.Devices.Client;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace DNSensorAzureIoTHub
+{
+    public class Program
+    {
+
+        // set up IoT Hub message
+        const string DeviceID = "<replace-with-your-device-id>";
+        const string IotBrokerAddress = "<replace-with-your-iot-hub-name>.azure-devices.net";
+
+        public static void Main()
+        {
+            Console.WriteLine($".Net IoT with BMP280 Sensor!");
+
+            // set up for LED and pin
+            const int pin = 18;
+            const int lightTime = 1000;
+            const int dimTime = 2000;
+            using GpioController led = new();
+            led.OpenPin(pin, PinMode.Output);
+    
+            // setup for BMP280
+            const int busId = 1;
+            I2cConnectionSettings i2cSettings = new(busId, Bmp280.DefaultI2cAddress);
+            I2cDevice i2cDevice = I2cDevice.Create(i2cSettings);
+            using var i2CBmp280 = new Bmp280(i2cDevice);
+
+            // Create an X.509 certificate object.
+            var cert = new X509Certificate2($"{DeviceID}.pfx", "1234");
+            var auth = new DeviceAuthenticationWithX509Certificate(DeviceID, cert);
+            DeviceClient azureIoT = DeviceClient.Create(IotBrokerAddress, auth, TransportType.Amqp_Tcp_Only);
```

Comment:
Why are you using AMQP?

Reply:
There's no reason to pick this specifically, I changed to MQTT protocol